### PR TITLE
Update members.yml

### PR DESCRIPTION
**Pull request by force2b**
Add my image and bio

### CODE REVIEW

**Comment by dextermilo** [comment]
@force2b hey, did Jon review your bio?

**Comment by dextermilo** [comment]
@jvarese

**Comment by force2b** [comment]
@dextermilo Actually no. I'm more than happy with any changes that @jvarese wants to make. Also, I used a slightly different profile pic than the standard github one that I have.

Also, I have to figure out how to get emails related to the SFdO repo's to go to my salesforce email. It should be possible through custom routing, but I might have to wait to be granted access to the org.

**Comment by force2b** [comment]
Though, it looks like github doesn't actually save each image so now it's using my more informal image. I'll just change the profile pic.

**Comment by njjc** [comment]
@dextermilo String review is done, thanks!

File: _data/members.yml [hunk 1]
```diff
@@ -206,7 +206,7 @@ Michael Smith:
   github:         force2b
   avatar:         https://avatars3.githubusercontent.com/u/801017?v=3&u=7428ed27e8017a1d72abf081351251c03a35807d&s=400
   title:          "Senior Member of the Technical Staff"
-  bio:            "I'm a 6x certified Salesforce Developer working on the platform since 2001. Four years after implementing Salesforce for a non-profit in 2005, I began consulting with other non-profits in the Salesforce space and quickly became hooked on the Non-Profit Success Pack and what it can do for non-profit organizations using Salesforce. Starting in 2017 as part of the NGO Development Team I look forward to further expanding the power and reach of the NPSP product line."
+  bio:            "I'm a 6x certified Salesforce Developer and have been working on the platform since 2001. Four years after implementing Salesforce for a nonprofit in 2005, I began consulting with other nonprofits and quickly became hooked on the Nonprofit Success Pack and what it can do for nonprofits using Salesforce. In 2017, I joined the Salesforce.org Development Team to help further expand the power and reach of the NPSP product line."
```
Comment (force2b) [inline _data/members.yml]: @smarovich So much better!

Comment (njjc) [inline _data/members.yml]: She's a real pro! ;)

**Comment by jvarese** [comment]
Guys - really sorry. I never got notification on this one, and am only seeing it now because Dylan pointed me to it. @force2b - where's your bio?

**Comment by force2b** [comment]
@jvarese The bio (which includes more than just changes I made) can be found in the Files tab: https://github.com/SalesforceFoundation/salesforcefoundation.github.io/pull/6/files

**Comment by force2b** [comment]
@jvarese test

**Comment by jvarese** [comment]
@force2b - how's this?

I'm a 6x certified Salesforce Developer and have been working on the platform since 2001. While consulting with and implementing Salesforce for various nonprofits I quickly became hooked on the Nonprofit Success Pack. In 2017, I joined the Salesforce.org Development Team to help further expand the power and reach of the NPSP product line.

**Comment by force2b** [comment]
@jvarese Thanks. It's so sad that I don't know how many certs I actually have. Excluding "Force.com Developer" (which I transitioned out of, but is still listed), I actually have 7 active certs - http://certification.salesforce.com/certification-detail-print?conId=003G0000017E2o2IAC. Other than that, it's great. Thanks!

**Comment by jvarese** [comment]
@njjc - I don't know what the process is. Am I actually supposed to merge this or someone else?

**Comment by njjc** [comment]
@jvarese : @dextermilo will merge

**Comment by dextermilo** [comment]
Oh I will, will I?

**Pull request merged**